src/utils/releases.rs

```diff
@@ -85,7 +85,14 @@ pub fn infer_gradle_release_name(path: Option<PathBuf>) -> Result<Option<String>
 
 /// Detects the release name for the current working directory.
 pub fn detect_release_name() -> Result<String> {
-    // cordova release detection first.
+    // try SENTRY_RELEASE environment variable
```

Comment:
if the comment is almost the same as the code below it, just leave out the comment